internal/replacements/replacements.go

```diff
@@ -200,14 +201,23 @@ func processGroupNeedsRemoval(cluster *fdbv1beta2.FoundationDBCluster, pod *core
 	}
 
 	if cluster.NeedsReplacement(processGroupStatus) {
-		specHash, err := internal.GetPodSpecHash(cluster, processGroupStatus.ProcessClass, idNum, nil)
+		spec, err := internal.GetPodSpec(cluster, processGroupStatus.ProcessClass, idNum)
```

Comment:
Oh, you are invoking this because you wanted to log "spec.String()" below.

Reply:
Yes, we had a few cases in the past where it would have been useful to see what the new spec looks like. Using the spec in json format and encoding it to base64 seems like the best solution to log the new desired spec (to prevent any parsing issues with the log forwarders).